frontend/src/components/MonacoEditor/sqlParser.ts

```diff
@@ -42,6 +46,32 @@ export const isMultipleStatements = (data: ParseResult["data"]) => {
   return isArray(data) && data.length > 1;
 };
 
+export const isDDLStatement = (data: ParseResult["data"]) => {
```

Comment:
How do we handle the case when the statements are a combination of DDL+DMLs?

Reply:
We parse the SQL statement, if multiple SQL statements are detected, we only execute the first statement.

Reply:
This may surprise users that the change or result doesn't match their expectation. We may want to warn or fail early for this case.